image-rs/src/config.rs

```diff
@@ -126,6 +127,14 @@ pub struct ImageConfig {
     #[serde(default = "Option::default")]
     pub registry_configuration_uri: Option<String>,
 
+    /// Registry configuration supports registry blocking, mirroring and remapping rules. 
+    /// This field points to a registry configuration file, which can either be stored locally
+    /// in the rootfs or retrieved from initdata.
+    ///
+    /// This value defaults to `None`.
+    #[serde(default = "Option::default")]
+    pub registry_config: Option<RegistryConfig>,
```

Comment:
It would do great help to add some unit tests for parsing the config. Could you add some near [here](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/hub/src/config.rs#L196-L203)?

Reply:
thx Ding, tests added